src/pysdmx/model/code.py

```diff
@@ -75,30 +77,9 @@ class Codelist(Struct, frozen=True, omit_defaults=True):
         codes: The list of codes in the codelist.
     """
 
-    id: str
-    name: str
-    agency: str
-    description: Optional[str] = None
-    version: str = "1.0"
-    codes: Sequence[Code] = ()
+    items: Sequence[Code] = ()
```

Comment:
We need to align the documentation "codes" with the implementation "items".
As discussed, my preference is to keep the "codes", but I am happy otherwise.

Reply:
Yes, I am still struggling a lot with this specific change, as many tests fail and need to modify the whole process, I will probably try to rollback this change and use just codes in it, as this was just to align with the IM.